app/webpack/stylesheets/_multi_file_upload.scss

```diff
@@ -0,0 +1,3 @@
+.moj-multi-file__uploaded-files:has(.moj-multi-file-upload__list:empty) {
```

Comment:
Nice. Does this also be used allow `app/webpack/javascripts/modules/Modules.Messaging.js` and `app/webpack/javascripts/modules/Modules.MultiFileUpload.js` to be cleaned up, where they show and hide the error box in javascript?

Also (sorry for my lack of knowledge of the finer details of CSS) is there a way to inherit from the `govuk-visually-hidden` class?

Reply:
Yes, I think we can reuse this to replace the javascript in those files. But I'm having issues getting that error box to show at all - I'm not sure if it's a problem with the `uploadFileErrorHook` or just me doing something wrong, but rather than complicate this ticket more I'll tackle that separately.

On your second point, we can do this
```
.moj-multi-file__uploaded-files:has(.moj-multi-file-upload__list:empty) {
  @extend .govuk-visually-hidden
}
```
to use the `govuk-visually-hidden` class, instead of explicitly setting the styling ourselves, which I prefer so I've pushed an update to that commit.